docs/contributing.rst

```diff
@@ -110,6 +110,8 @@ Do not use an autoformatter like :code:`black` as the configuration settings for
 might be incompatible with the changes made by :code:`black`. This is discussed in detail at
 `this link <https://docs.astral.sh/ruff/formatter/black/>`_.
 
+.. code-block:: bash
```

Comment:
It might be good to add more context surrounding this line instead of putting it directly after a discussion on another topic. Otherwise it seems like an orphaned line here.

Reply:
Got it , will work on it